### PR TITLE
Monitor certificate status

### PR DESCRIPTION
**Pull request by jenting**
# Make sure it's tested
*Technical writers will not always be able to verify the implementation!*

Any PR opened is assumed to have been verified by QA if not designated by comments or labels otherwise. Please make sure you have tested the changes and included any information that a user would require to use the documentation.

# Describe your changes

Use cert-exporter to exposes cluster-wide certificates status and use monitoring stack (Prometheus & Grafana) to receives alerts by Prometheus Alertmanager & monitors certificate status by Grafana dashboard.

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to: https://github.com/SUSE/avant-garde/issues/1488

# Enable maintainer updates
Please enable maintainer updates so we can push commits into your branch to make collaboration and reviews easier.

# Do not force push your branch
Please avoid force pushing to branches that are subject of pull requests. Force pushing breaks maintainer commits in many cases and is very hard (if not impossible) to untangle for backporting.

# Labels
Please set any (and all) appropriate labels that describe the status of the PR.

| **Label** | **Description** |
| --- | --- |
| P1 | PR should be worked on and merged as soon as possible |
| Blocked | Work can not proceed because other work has not been completed, PR can not be merged (code has not been merged but documentation is ready) |
| On-Hold | Underlying work is completed but the PR should not be merged |
| ReleaseNotes | User interaction is required after the introduction of this change and the change must be mentioned in the release notes |
| v3/v4/v4.x | Which version of the release the PR should be merged into, this can be multiple versions, please set the "Backport" label if it needs to go into a previous release |
| Needs Review | Some details of the PR are known to be incomplete and must be discussed with other engineers before merging (if possible assign reviewers or cc mention in comments), PR can not be merged |

### CODE REVIEW

File: adoc/admin-security-certificates.adoc [hunk 1]
```diff
@@ -145,6 +145,93 @@ The following certificates are created by `skuba`:
...
+----
+
+=== Prerequisites
+
+. To monitor the certificates, we need monitoring stack, follow the <<monitoring_stack>> on how to deploy it.
+. Label the skuba addon certificates
++
+[source,bash]
+----
+kubectl label --overwrite secret oidc-dex-cert -n kube-system caasp.suse.com/skuba-addon=true
```
[the remark this answers is not in the document]
Comment (jenting) [reply adoc/admin-security-certificates.adoc:165]: I think the answer is _yes_. In Secret resource `metadata.ownerReferences`.

**Comment by r0ckarong** [comment]
@jenting If this can be merged please remove the On-Hold label, otherwise I will wait for further discussion.

**Comment by jenting** [comment]
> @jenting If this can be merged please remove the On-Hold label, otherwise I will wait for further discussion.

You could review now :)
We are working on reviewing packaging container image, helm chart, and test cases.

**Comment by r0ckarong** [comment]
> > @jenting If this can be merged please remove the On-Hold label, otherwise I will wait for further discussion.
> 
> You could review now :)
> We are working on reviewing packaging container image, helm chart, and test cases.

Review done, please let me know when this can be merged.

**Comment by jenting** [comment]
> > > @jenting If this can be merged please remove the On-Hold label, otherwise I will wait for further discussion.
> 
> > 
> 
> > You could review now :)
> 
> > We are working on reviewing packaging container image, helm chart, and test cases.
> 
> 
> 
> Review done, please let me know when this can be merged.

okay, waits @cclhsu signal.

File: adoc/admin-monitoring-stack.adoc [hunk 2]
```diff
@@ -401,7 +404,9 @@ alertmanagerFiles:
       email_configs:
       - to: 'admin@example.com'
 ----
-. Replace the empty set of rules `rules: {}` in the `serverFiles` section of the configuration file.
+. Configures alerting rules in Prometheus server
++
+Replace the empty set of rules `rules: {}` in the `serverFiles` section of the Prometheus configuration file `prometheus-config-values.yaml`.
```
Comment (c3y1huang) [inline adoc/admin-monitoring-stack.adoc:408]:
```suggestion
Replace the `serverFiles` section of the Prometheus configuration file `prometheus-config-values.yaml`.
```

**Comment by c3y1huang** [comment]
@maximenoel8 could you help with testing this?

**Comment by c3y1huang** [comment]
So far it worked of me following the doc using `registry.suse.de/devel/caasp/4.0/containers/cr/containers/caasp/v4/cert-exporter`.
grafana-cert-exporter.

Something found during the run through: 
iIn `doc-caasp/adoc/admin-monitoring-stack.adoc`, Prerequisites section:
namespace should go before creating secret.
Screenshot-2020-04-23-18:17:29

**Comment by jenting** [comment]
> Something found during the run through:
> In `doc-caasp/adoc/admin-monitoring-stack.adoc`, Prerequisites section:
> namespace should go before creating secret.
> Screenshot-2020-04-23-18:17:29

Thanks, move the create namespace monitoring before create secret.

**Comment by r0ckarong** [comment]
@cclhsu Could you do the final review and confirm this for merge please? Thanks!

**Pull request merged**